### PR TITLE
Fix image height auto not being honoured due to container with display flex

### LINKED ISSUE #6380: Invite your team image is squished
## Repro

Setup a blog, and on step two you'll see a squished image:

screen shot 2016-01-25 at 08 15 28

## Specs

- latest master `e5e375057e7b8ed0ec797775a962553a57f18588`
- chrome 48

### PR DESCRIPTION
**Pull request by jamesslock**
closes #6380
- Set image dimensions on img tag for rendering performance
- Set image class to display block and height auto
- Remove display flex on container, not required.

### CODE REVIEW

**Comment by acburdine** [comment]
Looks good to me, but can you squash your commits down into one commit?

File: core/client/app/templates/setup/three.hbs [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <p>Ghost works best when shared with others. Collaborate, get feedback on your posts &amp; work together on ideas.</p>
 </header>
 
-<img class="gh-flow-faces" src="{{gh-path 'admin' 'img/users.png'}}" alt="" />
+<img class="gh-flow-faces" src="{{gh-path 'admin' 'img/users.png'}}" alt="" width="1716px" height="236px" />
```
Comment (kevinansfield) [inline core/client/app/templates/setup/three.hbs]: `width` and `height` attributes on elements shouldn't have the `px` on them

Comment (jamesslock) [inline core/client/app/templates/setup/three.hbs]: Fixed :)

**Comment by acburdine** [comment]
:+1:

**Comment by kevinansfield** [comment]
Closing in favour of #6427 as discussed in #6380. Sorry @jamesslock, I'll try not to snipe your PRs next time!

**Pull request closed without merging**